### PR TITLE
mv es-blockly-games to iiab rsync storage in order to rename

### PR DESCRIPTION
**Pull request by tim-moody**

### CODE REVIEW

**Comment by tim-moody** [comment]
needs testing

**Comment by holta** [comment]
Likely EN & FR should be streamlined in this fashion too?

**Comment by tim-moody** [comment]
The proper way to move es-blockly-games would have been to add it as a catalog fragment to https://github.com/iiab-share/iiab-content/blob/main/catalogs/iiab-module-cat.json and remove from the oer2go catalog, but it occupies a middle ground between the two catalogs and the oer2go catalog had to be changed anyway, so took the simpler path.

**Comment by tim-moody** [comment]
>Likely EN & FR should be streamlined in this fashion too?

don't see the benefit as no renaming is required and these two work fine.

File: docs/1.Overview.md [hunk 1]
```diff
@@ -33,3 +33,7 @@ The code can be found in the git repo https://github.com/iiab/iiab-admin-console
 Globally shared menu definitions are in https://github.com/iiab-share/js-menu-files.
 
 Catalog information is in https://github.com/iiab-share/iiab-content.
+
+## Document Version
+
+This document was last maintaineed on December 29, 2022.
```
Comment (holta) [inline docs/1.Overview.md:39]:
```suggestion
This document was last maintained on December 29, 2022.
```

File: docs/2.CMDSRV.md [hunk 2]
```diff
@@ -100,3 +100,7 @@ When running as a service CMDSRV traps all error, and this can mask problems tha
 This will start CMDSRV from the command line and allow python errors not to be trapped.
 
 iiab-cmdsrv-ctl 'COMMAND {"key": "value"}' can be used to test a particular Command. (Note use of single quotes to make the Command and parameters a single argument.)
+
+## Document Version
+
+This document was last maintaineed on December 29, 2022.
```
Comment (holta) [inline docs/2.CMDSRV.md:106]:
```suggestion
This document was last maintained on December 29, 2022.
```

File: docs/3.Console.md [hunk 3]
```diff
@@ -49,4 +49,8 @@ Wiki content comes in a growing list of languages. In order to translate languag
 
 # Troubleshooting
 
-The best way to see under the hood is to open the Javascript Console in your browser. Many functions write status to the console that can be examined.
\ No newline at end of file
+The best way to see under the hood is to open the Javascript Console in your browser. Many functions write status to the console that can be examined.
+
+## Document Version
+
+This document was last maintaineed on December 29, 2022.
```
Comment (holta) [inline docs/3.Console.md:56]:
```suggestion
This document was last maintained on December 29, 2022.
```

File: docs/4.JS-Menu.md [hunk 4]
```diff
@@ -32,3 +32,7 @@ The second icon makes the output less verbose. This is aimed at users who have b
 The third icon leads to a feedback form which will store comments in a local database.
 
 Choices made in the first two cases are particular to the user, or at least the device on which the user is operating.
+
+## Document Version
+
+This document was last maintaineed on December 29, 2022.
```
Comment (holta) [inline docs/4.JS-Menu.md:38]:
```suggestion
This document was last maintained on December 29, 2022.
```

File: docs/5.Content.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+The module should have a unique module_id. By convention we have been using IIAB1XX where 1XX is the next 3 digit number after the last item in the catalog.
+
+Manually add the catalog fragment to https://github.com/iiab-share/iiab-content/blob/main/catalogs/iiab-module-cat.json.
+
+As was the case with custom ZIMS you will need to create a Menu Item Definition for your modules.
+
+## Document Version
+
+This document was last maintaineed on December 29, 2022.
```
Comment (holta) [inline docs/5.Content.md:79]:
```suggestion
This document was last maintained on December 29, 2022.
```

**Comment by holta** [comment]
Can assets (catalogs, etc) that are no longer used be marked with a `.unused` suffix within https://github.com/iiab/iiab-admin-console/tree/master/assets ?

**Comment by tim-moody** [comment]
>Can assets (catalogs, etc) that are no longer used be marked with an .unused suffix within https://github.com/iiab/iiab-admin-console/tree/master/assets ?

sure. which ones are no longer used?

**Comment by tim-moody** [comment]
rm /opt/admin and did fresh install. nothing complained

**Pull request merged**